### ISSUE TITLE
 [Feature Request] Move through the document while holding space

### ISSUE THREAD

**Issue by SeeDborg** [OpenBoard-org/OpenBoard#591]
I think all is in the title :)
It would be a nice feature, it's present in many softwares nowadays.

**Comment by SeeDborg** [comment]
Wow, I see that Openboard 1.6.4 add nice features to navigate and zoom through a document. So I up this topic since I think it's related and it would help a lot to navigate.

**Comment by letsfindaway** [comment]
> I think all is in the title :)

Actually, for me not really. Do you mean you want to switch to the next page by pressing the space bar? Or do you have some other action in mind?

If you want to switch pages by using the keyboard, then you can use the left and right arrow keys.

**Comment by SeeDborg** [comment]
No, I don't mean moving from one page to another, but moving through the current page while I hold space and clic at the same time. Like in pdf or in photoshop. Here is an example :
![Animation](https://user-images.githubusercontent.com/7523821/194781097-92abdf9c-b1ad-4d20-bfc8-cb6fa9fb56d4.gif)

**Comment by kaamui** [comment]
It's already possible with the "hand" tool

**Comment by SeeDborg** [comment]
No.
What I describe is : 
 - Maintaining space bar is switching to the hand tool as long as I hold it.
 - Releasing the space bar is going back to the previous tool.

You can try it in any version of adobe reader, adobe photoshop, and many other softwares.
I think the feature is named something like : hold space to drag and pan.

**Comment by SeeDborg** [comment]
A similar feature that has just been added to OpenBoard would be : https://github.com/OpenBoard-org/OpenBoard/pull/343#event-7569129967

**Comment by kaamui** [comment]
OK. It would be indeed a nice feature, in terms of user experience. I'l try to ask the team if we add it to the todo list, but be aware that the OpenBoard's team is very small and that such a feature won't be a priority at all, for now. 

If you are willing to try to develop it, please provide a pull request and I would be glad to accept it.

**Comment by SeeDborg** [comment]
Thanks.
Sadly I cannot help, I know my way in php but not in C++ :(

**Comment by SeeDborg** [comment]
Another idea : same option but with middle mouse click :)
